src/main/java/seedu/opus/model/task/UniqueTaskList.java

```diff
@@ -97,12 +102,21 @@ public void setTasks(List<? extends ReadOnlyTask> tasks) throws DuplicateTaskExc
 
     public UnmodifiableObservableList<Task> asSortedList(String keyword) {
         switch (keyword) {
+        case "all":
+            FXCollections.sort(internalList, new TaskComparator());
+            break;
         case "priority":
             FXCollections.sort(internalList, new PriorityComparator());
             break;
-        case "deadline":
+        case "start":
+            FXCollections.sort(internalList, new StartTimeComparator());
+            break;
+        case "end":
             FXCollections.sort(internalList, new EndTimeComparator());
             break;
+        case "status":
+            FXCollections.sort(internalList, new StatusComparator());
+            break;
```

Comment:
These commands in strings should be declared as a `static` variables in the `SortCommand` class, so we don't have to look through the entire codebase to change the behaviors of the command. 

Maybe name them `BY_ALL`, `BY_PRIORITY` etc.